[WPE] Unreviewed test gardening

dpino opened the pull request:
#### 5cd5802e10c8a2b6b7a545377a486f4ede2b39d0
<pre>
[WPE] Unreviewed test gardening

Update tests expectations and baselines.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/compositing/layer-creation/will-change-layer-creation-expected.txt:
* LayoutTests/platform/glib/css3/filters/backdrop/backdrop-filter-does-not-size-properly-absolute-expected.txt:
* LayoutTests/platform/glib/fast/multicol/table-vertical-align-expected.txt:
* LayoutTests/platform/glib/fast/ruby/ruby-run-break-expected.png:
* LayoutTests/platform/glib/fast/ruby/ruby-run-break-expected.txt:
* LayoutTests/platform/glib/fast/table/024-expected.txt:
* LayoutTests/platform/glib/fast/table/tableInsideCaption-expected.txt:
* LayoutTests/platform/glib/fast/table/vertical-align-baseline-expected.txt:
* LayoutTests/platform/glib/fast/text/text-letter-spacing-expected.png:
* LayoutTests/platform/glib/fast/text/text-letter-spacing-expected.txt:
* LayoutTests/platform/glib/fast/writing-mode/Kusa-Makura-background-canvas-expected.txt:
* LayoutTests/platform/glib/http/wpt/mediarecorder/MediaRecorder-first-frame-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/the-iframe-element/iframe_sandbox_anchor_download_block_downloads.tentative-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-navigations-in-multiple-windows-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/bugs/bug10296-1-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tbody_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tfoot_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/thead_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tr_valign_baseline-expected.txt:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/fast/css/focus-ring-exists-for-search-field-expected.png: Added.
* LayoutTests/platform/wpe/fast/css/focus-ring-exists-for-search-field-expected.txt:
* LayoutTests/platform/wpe/fast/mediastream/captureStream/canvas3d-expected.txt:
* LayoutTests/platform/wpe/fast/multicol/table-vertical-align-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/024-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/tableInsideCaption-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/vertical-align-baseline-expected.png: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-table-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-td-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/caption-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/colgroup-col-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/table-quirks-expected.txt:
* LayoutTests/platform/wpe/tables/mozilla/bugs/bug10296-1-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tbody_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tfoot_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/thead_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tr_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla_expected_failures/bugs/bug2479-5-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla_expected_failures/bugs/bug2479-5-expected.txt:

Canonical link: <a href="https://commits.webkit.org/287338@main">https://commits.webkit.org/287338@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/cd8c783d80e2b6ce4a80f1a92b74ea87438a4a92

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/79245 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/58273 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/32617 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/83868 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/30412 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/67365 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/6544 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/61992 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/19892 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/82312 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/52052 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/72203 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/42298 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/49397 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/26365 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/28800 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/70509 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/26801 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/85262 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/6543 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/4537 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/70238 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/6706 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/68075 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/69486 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/13538 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/12370 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/12230 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/6500 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/6422 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/9889 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/8214 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b7cb2c7dfa4b26260db44beda0b99ae6a33ddfd0. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/37415

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/fade53c562fe3b93c30567b62d6c384c5d8aa954. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/37415

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/cd8c783d80e2b6ce4a80f1a92b74ea87438a4a92. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/37415

dpino committed:
[WPE] Unreviewed test gardening

Update tests expectations and baselines.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/compositing/layer-creation/will-change-layer-creation-expected.txt:
* LayoutTests/platform/glib/css3/filters/backdrop/backdrop-filter-does-not-size-properly-absolute-expected.txt:
* LayoutTests/platform/glib/fast/multicol/table-vertical-align-expected.txt:
* LayoutTests/platform/glib/fast/ruby/ruby-run-break-expected.png:
* LayoutTests/platform/glib/fast/ruby/ruby-run-break-expected.txt:
* LayoutTests/platform/glib/fast/table/024-expected.txt:
* LayoutTests/platform/glib/fast/table/tableInsideCaption-expected.txt:
* LayoutTests/platform/glib/fast/table/vertical-align-baseline-expected.txt:
* LayoutTests/platform/glib/fast/text/text-letter-spacing-expected.png:
* LayoutTests/platform/glib/fast/text/text-letter-spacing-expected.txt:
* LayoutTests/platform/glib/fast/writing-mode/Kusa-Makura-background-canvas-expected.txt:
* LayoutTests/platform/glib/http/wpt/mediarecorder/MediaRecorder-first-frame-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/the-iframe-element/iframe_sandbox_anchor_download_block_downloads.tentative-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-navigations-in-multiple-windows-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/bugs/bug10296-1-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tbody_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tfoot_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/thead_valign_baseline-expected.txt:
* LayoutTests/platform/glib/tables/mozilla/marvin/tr_valign_baseline-expected.txt:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/fast/css/focus-ring-exists-for-search-field-expected.png: Added.
* LayoutTests/platform/wpe/fast/css/focus-ring-exists-for-search-field-expected.txt:
* LayoutTests/platform/wpe/fast/mediastream/captureStream/canvas3d-expected.txt:
* LayoutTests/platform/wpe/fast/multicol/table-vertical-align-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/024-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/tableInsideCaption-expected.png: Added.
* LayoutTests/platform/wpe/fast/table/vertical-align-baseline-expected.png: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-table-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-td-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/caption-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/colgroup-col-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/table-quirks-expected.txt:
* LayoutTests/platform/wpe/tables/mozilla/bugs/bug10296-1-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tbody_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tfoot_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/thead_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla/marvin/tr_valign_baseline-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla_expected_failures/bugs/bug2479-5-expected.png: Added.
* LayoutTests/platform/wpe/tables/mozilla_expected_failures/bugs/bug2479-5-expected.txt:

Canonical link: https://commits.webkit.org/287338@main
